### PR TITLE
Update default-infrastructure-monitoring-data.mdx

### PR DESCRIPTION
**Pull request by nr-ksteinbach**
feat: document the improvement in infrastructure aggregation, that is slowly rolled out in the October and November 2024 period

(details - see slides linked from NR-278919)

### CODE REVIEW

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx [hunk 1]
```diff
@@ -177,6 +177,10 @@ You can [query your infrastructure data](/docs/using-new-relic/data/understand-d
 SELECT * FROM ProcessSample
 ```
 
+Metrics from the New Reic Infrastructure Agent (SystemSample, StorageSample, NetworkSample and ProcessSample) are aggregated into buckets, in order to improve the query performance over longer periods of time.
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx:180]:
```suggestion
Metrics from the New Relic infrastructure agent (`SystemSample`, `StorageSample`, `NetworkSample` and `ProcessSample`) are aggregated into buckets to improve the query performance over longer periods of time.
```

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx [hunk 2]
```diff
@@ -177,6 +177,10 @@ You can [query your infrastructure data](/docs/using-new-relic/data/understand-d
...
+NRQL queries for under 1 hour of time will scan the raw data, and longer queries will scan 1 minute / 10 minute / 1 hour and 3 hour aggregated buckets.
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx:181]:
```suggestion
NRQL queries scoped to under 1 hour of time will scan the raw data. For queries scoped over 1 hour, data will be scanned in the following aggregated buckets:
*  1 minute
*  10 minute
*  1 hour
*  3 hours
```

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx [hunk 3]
```diff
@@ -177,6 +177,10 @@ You can [query your infrastructure data](/docs/using-new-relic/data/understand-d
...
+Until October/November 2024 these aggregated event buckets only contained 1 average value per metric. Starting in October/November 2024 the aggregation has been improved to use the [gauge format](/docs/data-apis/understand-data/metric-data/metric-data-type/) for accurate differentiation between `max(), min() and average()`.
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx:182]: @nr-ksteinbach Can you specify dates here? If not, I'd prefer to say "Until November 2024..."

Comment (ally-sassman) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx:182]:
```suggestion
Until October/November 2024 these aggregated event buckets only contained 1 average value per metric. Starting in October/November 2024 the aggregation has been improved to use the [gauge format](/docs/data-apis/understand-data/metric-data/metric-data-type/) for accurate differentiation between `max()`, `min()`, and `average()`. 
```

Comment (nr-ksteinbach) [reply src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx:186]: answered in slack.

**Review by nr-ksteinbach — COMMENTED** [review]
reviewed / second round (thank you, Ally!)

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx [hunk 4]
```diff
@@ -177,6 +177,14 @@ You can [query your infrastructure data](/docs/using-new-relic/data/understand-d
 SELECT * FROM ProcessSample
 ```
 
+Metrics from the New Relic infrastructure agent (`SystemSample`, `StorageSample`, `NetworkSample` and `ProcessSample`) get aggregated into buckets to improve the query performance over longer periods of time.
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx:180]:
```suggestion
Metrics from the New Relic infrastructure agent (`SystemSample`, `StorageSample`, `NetworkSample`, and `ProcessSample`) get aggregated into buckets to improve the query performance over longer periods of time.
```

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx [hunk 5]
```diff
@@ -177,6 +177,14 @@ You can [query your infrastructure data](/docs/using-new-relic/data/understand-d
 SELECT * FROM ProcessSample
 ```
 
+Metrics from the New Relic infrastructure agent (`SystemSample`, `StorageSample`, `NetworkSample`, and `ProcessSample`) get aggregated into buckets to improve the query performance over longer periods of time.
+NRQL queries scoped to under 1 hour of time will scan the raw data. For queries scoped over 1 hour, data will be scanned in the following aggregated buckets:
+*  1 minute (scopes of 1 hour to 6 hours)
+*  10 minutes (scopes of 6 hours to 3 days)
+*  1 hour (scopes of 3 days to 14 days)
+*  3 hours (scopes of 14+ days)
+Until October/November 2024 these aggregated event buckets only contained 1 average value per metric. Starting in October/November 2024, we've improved the aggregation to use the [gauge format](/docs/data-apis/understand-data/metric-data/metric-data-type/) for correct differentiation between `max()`, `min()`, and `average()` values.
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx:186]:
```suggestion
Until October/November 2024 these aggregated event buckets only contained 1 average value per metric. Starting in October/November 2024, we've improved the aggregation to use the [gauge format](/docs/data-apis/understand-data/metric-data/metric-data-type/) for correct differentiation between `max()`, `min()`, and `average()` values. 
```
```suggestion
Historically, these aggregated event buckets only contained one average value per metric. During October and November 2024, we're gradually migrating the aggregation to use the [gauge format](/docs/data-apis/understand-data/metric-data/metric-data-type/) instead. This allows you to differentiate between `max()`, `min()`, and `average()` values. 
```

Comment (ally-sassman) [reply src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx:186]: @nr-ksteinbach What do you think of this revision? Also, how can customers check if their metrics have been migrated?

**Review by nr-ksteinbach — COMMENTED** [review]
looks good to me. Many thanks Ally for your help.

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx [hunk 6]
```diff
@@ -177,8 +177,28 @@ You can [query your infrastructure data](/docs/using-new-relic/data/understand-d
 SELECT * FROM ProcessSample
 ```
 
+Metrics from the New Relic infrastructure agent (`SystemSample`, `StorageSample`, `NetworkSample`, and `ProcessSample`) get aggregated into buckets to improve the query performance over longer periods of time.
+NRQL queries scoped to under 1 hour of time will scan the raw data. For queries scoped over 1 hour, data will be scanned in the following aggregated buckets:
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx:181]:
```suggestion
NRQL queries spanning less than one hour directly access the raw data. For queries exceeding one hour, data is retrieved from the following aggregated buckets:
```

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**